### DISCUSSION TITLE
CefSharp v113.3.50 nuget packages

### DISCUSSION THREAD

**Discussion by jozefizso** [cefsharp/CefSharp#4517] [Q&A]
Hi, I tried to update our project to [CefSharp v113.3.50 release](https://github.com/cefsharp/CefSharp/releases/tag/v113.3.50) but NuGet.org has the [release 113.1.40](https://www.nuget.org/packages/CefSharp.WinForms/113.1.40) listed as latest.

https://www.nuget.org/packages/CefSharp.WinForms#versions-body-tab

**Comment by amaitland** [comment]
Packages are on `Nuget.org`

Packages goto the myget feed first, so if someone hasn't been pushed to nuget.org you'll find it on https://www.myget.org/F/cefsharp/api/v3/index.json
